File: 3227. Vowels Game in a String/solution.cpp
#include <string>
using namespace std ; 
class Solution {
public:
    bool doesAliceWin(string s) {
        for( char c : s ){
            if( c=='a'|| c=='e'||c=='u'||c=='i'||c=='o' ){
                return true ; 
            }
        }
        return false ; 
    }
};

File: 3227. Vowels Game in a String/solution.java
class Solution {
    public boolean doesAliceWin(String s) {
        int vowels = 0 ; 
        for( char c : s.toCharArray() ){
            if( c=='a' || c=='e' || c=='i' || c=='a' || c=='o' || c=='u' ) vowels++ ; 
        }
        if(vowels!=0 ){
            return true ;
        }
        return false ; 
    }
}